client/data/settings/action-types.js

```diff
@@ -5,4 +5,7 @@ export default {
 	SET_IS_WCPAY_ENABLED: 'SET_IS_WCPAY_ENABLED',
 	SET_ENABLED_PAYMENT_METHOD_IDS: 'SET_ENABLED_PAYMENT_METHOD_IDS',
 	SET_IS_SAVING_SETTINGS: 'SET_IS_SAVING_SETTINGS',
+	SET_IS_MANUAL_CAPTURE_ENABLED: 'SET_IS_MANUAL_CAPTURE_ENABLED',
+	SET_ACCOUNT_STATEMENT_DESCRIPTOR_ENABLED:
```

Comment:
It looks like the action is called `SET_ACCOUNT_STATEMENT_DESCRIPTOR_ENABLED`, but it's not "enabling" a the account statement description (i.e.: it's not setting a true/false flag).
What do you think about calling it `UPDATE_ACCOUNT_STATEMENT_DESCRIPTOR`, like the action?

Reply:
Oh yes, good catch! :eagle: It wasn't my intention to use `_ENABLED` in the name of that var. That was just my copy&pasting and failing miserably renaming the var :smile: